ctap_keyring_device/ctap_keyring_device.py

```diff
@@ -79,6 +79,10 @@ def __init__(self):
             ctap2.Ctap2.CMD.GET_INFO: self.get_info,
         }
 
+        algorithms = []
+        for i in cose.CoseKey.supported_algorithms():
+            algorithms.append({'type': 'public-key', 'alg': i})
```

Comment:
I think `fido2.webauthn.PublicKeyCredentialType.PUBLIC_KEY` is a constant for `"public-key"`